Yogic.py

```diff
@@ -1 +1,441 @@
+        response = Response(json.dumps({'id':teacher_id}))
+    except:
+        response = Response(json.dumps({'error': 'Something went wrong,try again later'}))
+    response.headers['Content-type'] = 'application/json'
+    return response
+
+
+
+@app.route('/api/login/teacher', methods=['POST'])
+def login_teacher():
+    email = request.json.get('email','').lower()
+    password = request.json.get('password',None)
+    if not email or not password:
+        return jsonify({"msg": "Email or password invalid"}), 400
+
+
+    user = Teacher.query.filter_by(email=email).first()
+    if check_password(user.password_hash, email + password):
+        response = Response(json.dumps({'id':user.id}))
+        access_token = create_access_token(identity={'teacher':user.id})
+        response = Response(json.dumps({'access_token':access_token}))
+        response.headers['Content-type'] = 'application/json'
+        return response
+    else:
+        response=Response(json.dumps('wrong email or password'))
+        response.headers['Content-type'] = 'application/json'
+        return response
+
+
+
+@app.route('/api/teacher', methods=['PUT'])
+@jwt_required
+def edit_teacher():
+    claim = get_jwt_identity()
+    data = request.json
+    teacher_id = claim.get('teacher')
+    print(data, data.get('id', ''))
+    teacher = Teacher.query.filter_by(id=teacher_id) 
+    for _teacher in teacher:
+        _teacher.first_name=data.get('first_name')
+        _teacher.last_name=data.get('last_name')
+        _teacher.years_experience=data.get('years_experience')
+        _teacher.email=data.get('email')
+        
+    db.session.commit()
+    return jsonify({ }), 200
+
+    
+#signup form in React will call this method
+@app.route('/api/student', methods=['POST'])
+def create_student():
+    data = request.json
+    print(data)
+    new_student = Student(
+        first_name = data.get('first_name',''),
+        last_name = data.get('last_name', ''),
+        email = data.get('email', ''),
+        password_hash = set_password(data.get('email','').lower() + data.get('password', '')),
+    )
+    db.session.add(new_student)
+    db.session.commit()
+    student_id = (new_student.id)
+    return Response(json.dumps({'id':student_id}), mimetype='application/json')
+
+@app.route('/api/student', methods=['GET'])
+@jwt_required
+def get_student():
+    claim = get_jwt_identity()
+    student_id = claim.get('student')
+    print (student_id)
+    student = Student.query.filter_by(id=student_id)
+
+    list_student = []
+    for _student in student:
+        d_student = dict()
+        d_student.update({'id':_student.id})
+        d_student.update({'first_name': _student.first_name})
+        d_student.update({'last_name': _student.last_name})
+        list_student.append(d_student)
+    return json.dumps(list_student)
+
+@app.route('/api/bookclass', methods=['PUT'])
+@jwt_required
+def book():
+    api_key = 'e63418a078174be4d3ddd19b1b670397'
```

Comment:
look into moving to env file

Reply:
env file on Heroku isn't a good approach. Instead, you can use its built-in support for environment variables, using heroku config:set <var> <value> or its web UI. Either way, you'll get a regular environment variable.

Reply:
done